docs/UserGuide.md

```diff
@@ -234,28 +263,112 @@ Oops! There seems to be an error, please check your fields again.
-* Must be a positive integer 1, 2, 3, …
-* Multiple indexes are allowed for mass deletion,  each index separated by a white space.
-* `delete` without an index is valid if and only if there exists only one data entry in the current list.
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd fosterer in the address book
-* `find Jerry` followed by `delete 1` deletes the 1st fosterer in the results of the find command
+* `find Jerry` followed by `delete 1` deletes the 1st fosterer in the result list of the find command
 * `list` followed by `delete 1 3 7` deletes the 1st, 3rd and 7th fosterers in the address book
+* `list` followed by `delete 3 3 3 3` deletes the 3rd fosterer in the address book
+
 
 Expected output (success):
+`list` followed by `delete 1 3 7`
 ```agsl
-Fosterers Jerry Chee, John Doe, and Mary Ann are successfully deleted!
+3 fosterers deleted:
+Alex Yeoh; Phone: 87438807; Email: alexyeoh@example.com; Address: Blk 30 Geylang Street 29, #06-40; Housing: HDB; Availability: Available; Animal name: nil; Animal type: able.Cat; Tags: [new], 
+Charlotte Oliveiro; Phone: 93210283; Email: charlotte@example.com; Address: Blk 11 Ang Mo Kio Street 74, #11-04; Housing: HDB; Availability: Available; Animal name: nil; Animal type: able.Dog; Tags: [new], 
+Roy Balakrishnan; Phone: 92624417; Email: royb@example.com; Address: Blk 45 Aljunied Street 85, #11-31; Housing: Landed; Availability: Available; Animal name: nil; Animal type: able.Cat; Tags: [new]
 ```
 
 Expected output (fail):
+
+1. A non-positive index provided: `delete -1`
 ```agsl
-Oops! Invalid fosterer index provided, please check again.
+Invalid command format! 
+delete: Deletes fosterers identified by the index number used in the displayed person list.
+Parameters: INDEX [INDEX...] (must be a positive integer)
+Example: delete 1 2 3
 ```
+
+2. An index exceeds the number of fosterers in the list: `delete 0 1 99` but the list only has 6 fosterers
+```agsl
+The fosterer index provided is invalid
+```
+
+### Viewing statistics : `stats`
+Displays the requested statistic about the currently displayed list.
+
+Format: `stats FIELD`
+
+Parameters:`FIELD` 
+* The requested field whose statistic is to be calculated.
+* Valid fields that can be requested for are:
+  * `avail` :  shows the statistics of fosterers  who are available to foster, and the animals they can foster.
+  * `current` : shows the statistics of fosterers who are currently fostering, and the animals they are fostering.
+  * `housing` : shows the statistics of the various housing types.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+Statistics are calculated based on the currently displayed list. If you performed a `find available` command before `stats avail`, the resulting statistic will show that 100% of listed fosterers are available. Hence, please ensure that the current list for fosterers is the desired list you want your statistic to be calculated from. 
```

Comment:
Sounds more difficult to implement HAHA

Reply:
actually not really!!! because the model has a method to get the last shown list. another rationale is to give them more flexibility, eg they want to know out of the current list of fosterers that are available, how many stay in hdb! smt like that